guides/common/modules/ref_system-requirements.adoc

```diff
@@ -3,105 +3,60 @@
-* A minimum of 12 GB RAM is required for {SmartProxyServer} to function.
-In addition, a minimum of 4 GB RAM of swap space is also recommended.
-{SmartProxy} running with less RAM than the minimum value might not operate correctly.
+** {smart-proxy-minimum-memory} or higher
 endif::[]
-
-ifdef::katello,orcharhino,satellite[]
-* A unique host name, which can contain lower-case letters, numbers, dots (.) and hyphens (-)
-endif::[]
-
-ifdef::satellite[]
-* A current {ProjectName} subscription
-endif::[]
-* Administrative user (root) access
-* Full forward and reverse DNS resolution using a fully-qualified domain name
-
-{Project} only supports `UTF-8` encoding.
-If your territory is USA and your language is English, set `en_US.utf-8` as the system-wide locale settings.
-ifndef::foreman-deb[]
-For more information about configuring system locale in {EL}, see {RHELDocsBaseURL}9/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings[Configuring the system locale] in _{RHEL}{nbsp}9 Configuring basic system settings_.
-endif::[]
-
-ifdef::satellite[]
-Your {Project} must have the {SatelliteSub} manifest in your Customer Portal.
-{Project} must have {project-context}-{smart-proxy-context}-6.x repository enabled and synced.
-To create, manage, and export a Red{nbsp}Hat Subscription Manifest in the Customer Portal, see {RHDocsBaseURL}subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
-endif::[]
-
-{ProjectServer} and {SmartProxyServer} do not support shortnames in the hostnames.
-When using custom certificates, the Common Name (CN) of the custom certificate must be a fully qualified domain name (FQDN) instead of a shortname.
-This does not apply to the clients of a {Project}.
-
-Before you install {ProductName}, ensure that your environment meets the requirements for installation.
-ifeval::["{context}" == "{smart-proxy-context}"]
-[WARNING]
-====
-The version of {SmartProxy} must match with the version of {Project} installed.
-It should not be different.
-For example, the {SmartProxy} version {ProjectVersion} cannot be registered with the {Project} version {ProjectVersionPrevious}.
-====
+ifdef::katello,satellite[]
+** 4 GB RAM of swap space or higher
 endif::[]
 
-{ProductName} must be installed on a freshly provisioned system that serves no other function except to run {ProductName}.
-The freshly provisioned system must not have the following users provided by external identity providers to avoid conflicts with the local users that {ProductName} creates:
+** A unique host name, which can contain lower-case letters, numbers, dots (.) and hyphens (-)
 
-* {apache-user}
+* Ensure the system does not have the following users provided by external identity providers.
```

Comment:
I always thought this list is fairly long and most users won't have this. I've always wondered if an appendix with those names was better.